docker-compose.yml

```diff
@@ -112,7 +116,7 @@ services:
     command: ["bundle", "exec", "sidekiq","-c","2"]
 
   db:
-    image: postgres:11-alpine
+    image: postgres:13-alpine
```

Comment:
This is a breaking change. It's breaking development environments which can easily be reseeded - and it breaks by failing to start if you've run docker before with pg 11 (and a pretty clear error).

I think it's okay if the container docs are updated to point to this.